### DISCUSSION TITLE
Deployment to Google Cloud using Docker

### DISCUSSION THREAD

**Discussion by PMLS3** [capawesome-team/capacitor-plugins#132] [Q&A] [answered]
How can one deploy to Google Cloud using Docker.

I'm getting the following:  No authorization header was set for the request.
ERR_PNPM_FETCH_401  GET https://npm.pkg.github.com/download/@capawesome-team/capacitor-nfc/5.1.0

**Comment by robingenz** [comment, marked as answer]
Please take a look at our [Common issues](https://capawesome.io/sponsors/insiders/getting-started/#common-issues) and [CI/CD Configuration](https://capawesome.io/sponsors/insiders/getting-started/#cicd-configuration) section.
